src/components/ActionManager/ActionManager.js

```diff
@@ -0,0 +1,119 @@
+import Cosmos from "@lunie/cosmos-js"
+import config from "src/config"
+import { getSigner } from "./signer"
+import transaction from "./transactionTypes"
+
+import { formatCosmosArguments, convertCurrencyData } from "./cosmosMessages"
```

Comment:
I have the feeling `formatCosmosArguments` and what `cosmos-js` provides for messages is a dupe. Should we merge some of those conversions into `cosmos-js` or should we keep this separate?

Reply:
The reason I created these was the format is a little different. I believe i wanted to keep the Lunie options as a flat structure, and maintain the existing format. It does seem like the obvious choice to reuse the `cosmos-js` funcs now that I see it from afar.

I'll get back to you if that works without problem.